cypress/integration/pageobjects/inventory/view-lots-page.ts

```diff
@@ -0,0 +1,11 @@
+import { getIframeBody } from '../../../support/commands';
+
+export default class ViewLotsPage{
+    verifyLotDetailsDisplayed() {
+        getIframeBody().find('[data-test="lotDetailDialog"]').should('exist');
```

Comment:
Kindly add assertions to check if certain lot details are correct like GID or lot uuid if possible. Thanks!